containers/nginx-static-server/Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM nginx:latest
+
+# Set working directory to the Nginx document root
+WORKDIR /usr/share/nginx/html
+
+# Download the Terraform binary zip file, clean up and reduce the size of the Docker image by removing unnecessary files after installing packages.
+RUN apt-get update && apt-get install -y curl && \
```

Comment:
Please add this to a directory with the version name, ie download to 
`1.4.7/terraform_1.4.7_linux_amd64.zip`

Reply:
yup, Updated.
Infact updated it to a directory something like `terraform/1.4.7/terraform_1.4.7_linux_amd64.zip`